templates/test_only/index.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
```

Comment:
> Verta mokintis nepalikti šiukšlių.
> 
> Plačiau:
> * https://en.wikipedia.org/wiki/Technical_debt

Nes kolega turės perskaityti tavo **visą** pakeistą kodą.
Plačiau:
 * https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews